### PR TITLE
Java Gateway Update

### PR DESCRIPTION
**Pull request by pyrodie18**
##### SUMMARY
Initial changes to java gateway role for 2.0.0

##### COMPONENT NAME
Java Gateway Role

##### ADDITIONAL INFORMATION
Major Changes include:

- Updating Molecule matrix to test all OS, with all versions, and all databases (#777 )
- Updated documentation
- Removed support for anything < 5.0 (#766)
- Cleaned up handling of variables between OSs
- Added Ubuntu 22.04 (#763)
- Added Support for Debian 11 (#764)
- Removed support for EOL operating systems
- Cleaned up some variables (#768 )

### CODE REVIEW

File: changelogs/fragments/javagateway.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+minor_changes:
```
Comment (D3DeFi) [inline changelogs/fragments/javagateway.yml:1]: both of this sentences does not bring any value to users of this collection and I would recommend to remove them entirely

File: changelogs/fragments/javagateway.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  - github workflow - Updated workflow files to checked against all supported combinations
+  - documentation - Updated role documentation to reflect changes
+breaking_changes:
```
Comment (D3DeFi) [inline changelogs/fragments/javagateway.yml:4]: all changes here need to be prepended with role name otherwise resulting changelog will be a mess

**Review by D3DeFi — CHANGES_REQUESTED** [review]
See my comments related to changelogs, otherwise this is a good patch

File: changelogs/fragments/javagateway.yml (hunk 2 above)
Comment (pyrodie18) [reply changelogs/fragments/javagateway.yml]: Updates Made

**Review by D3DeFi — APPROVED** [review]

**Pull request merged**